views/Main.react.js

```diff
@@ -534,24 +513,6 @@ class MainPage extends Component {
       return index % 2 === 0 ? 'evenRow' : 'oddRow'
     }
   }
-  getAvatar = () => {
```

Comment:
Again we are finding ourselves in a position where we have to update similar block of code in multiple places! I see `getAvatar = ()` function in many places.

Reply:
Ignore it, sorry!

Reply:
That's why I removed the duplicated code from each view and pulled it out into its own module called `avatarUtil`.